wifiphisher/pywifiphisher.py

```diff
@@ -925,6 +925,9 @@ def run():
 
     time.sleep(3)
 
+    #no longer need mac_matcher
+    mac_matcher.unbind()
```

Comment:
I think if we do that having only one public method for an object is bit over excessive.